src/Services/CoreGameService.php

```diff
@@ -80,4 +80,8 @@ public function getModuleManager(): ModuleManager
     {
         return $this->game->getModuleManager();
     }
+
+    public function test() {
```

Comment:
Something that's no longer in there. I used this to test if the graphql-scheme "access" field correctly access that method. Removed it and changes the way to control access to fields anyway.